js/ui/html_editor/ui.html_editor.js

```diff
@@ -0,0 +1,254 @@
+import $ from "../../core/renderer";
+import { extend } from "../../core/utils/extend";
+import { isDefined } from "../../core/utils/type";
+import { getPublicElement } from "../../core/utils/dom";
+import registerComponent from "../../core/component_registrator";
+import EmptyTemplate from "../widget/empty_template";
+import Editor from "../editor/editor";
+
+import QuillRegistrator from "./quill_registrator";
+import DeltaConverter from "./converters/deltaConverter";
+import MarkdownConverter from "./converters/markdownConverter";
```

Comment:
I suppose markdownConverter should be imported manually by a customer not to pull redundant vendor dependencies